en/faq.md

```diff
@@ -0,0 +1,36 @@
+# FAQ - Frequently Asked Questions
+
+## What is Phalcon
+Phalcon in an open source full stack PHP framework written as a C language extension.
+
+## How Phalcon Works
+Phalcon is *not* an accelerator for PHP or projects written in PHP. 
```

Comment:
I think it might be nice if we mention Zephir here.

Phalcon is *not* an accelerator for PHP or projects written in PHP. _(For a PHP-like language to produce high-performance C extensions please see [Zephir](https://github.com/phalcon/zephir))_

Reply:
I agree